Features/links.feature

```diff
@@ -0,0 +1,529 @@
+    En bevat de link.self niet de tijdreisparameter geldigOp 
+    En bevat de link.self niet de tijdreisparameter beschikbaarOp
+
+    Als er een bevraging wordt gedaan op één van de endpoints (niet zijnde een levenscyclus endpoint)
+    En deze endpoint levert een collectie van resources
+    En een client geeft bepaalde parameters niet op (bv. expand maar niet geldigOp en beschikbaarOp)
+    Dan bevat _embedded van de collectie de resources in die collectie
+    En bevat de _links.self van iedere resource in de collectie een link naar het tijdreis endpoint van het betreffende resource type en met de resource ID, zoals hierboven beschreven (binnen het scenario)
+    En bevat de _links.self van iedere resource in de collectie deze parameters niet
+
+    Als er een bevraging wordt gedaan op één van de endpoints (niet zijnde een levenscyclus endpoint)
+    En deze endpoint levert een collectie van resources
+    En een client geeft bepaalde parameters niet op (bv. expand maar niet geldigOp en beschikbaarOp)
+    En de default waarde wordt gebruikt voor deze parameters
+    Dan bevat _embedded van de collectie de resources in die collectie
+    En bevat de _links.self van iedere resource in de collectie een link naar het tijdreis endpoint van het betreffende resource type en met de resource ID, zoals hierboven beschreven (binnen het scenario)
+    En bevat de _links.self van iedere resource in de collectie deze parameters niet
+
+  Scenario: self link van een embedded resource in een resource
+    Als er een bevraging wordt gedaan op één van de endpoints (niet zijnde een levenscyclus endpoint)
+    En deze endpoint levert één enkele resource
+    En één of meer gerelateerde resource worden geëxpand
+    Dan bevat _embedded de opgevraagde en aan de resource gerelateerde resources
+    En bevat de _links.self van een embedded resources in de resource een link die op dezelfde manier is samengesteld als bij het scenario 'self link van een resource in een collectie'
+
+    Als er een bevraging wordt gedaan op één van de endpoints (niet zijnde een levenscyclus endpoint)
+    En deze endpoint levert een collectie van resources
+    En één of meer gerelateerde resource worden geëxpand
+    Dan bevat _embedded van iedere resource in de collectie de opgevraagde en aan de resource gerelateerde resources
+    En bevat de _links.self van een embedded resources in de resource een link die op dezelfde manier is samengesteld als bij het scenario 'self link van een resource in een collectie'
+
+Functionaliteit: links naar gerelateerde resources
+
+  Scenario: links naar gerelateerde resources in een resource
+    Als er een bevraging wordt gedaan op één van de endpoints (niet zijnde een levenscyclus endpoint)
+    En deze endpoint levert één of meerdere resource
+    En de resource heeft geen relatie met andere resources
```

Comment:
Waarom is het eerste voorbeeld onder de kop van het scenario : links naar gerelateerde resources etc.  een voorbeeld van een object die geen relatie heeft. Voor mij is het vanzelfsprekend dat als er geen relatie is, er ook geen link is, want waarheen dan? Waarom hij misschien toch opgenomen moet worden, zou kunnen zijn om te verduidelijken dat het gaat om relaties volgens het informatiemodel BAG. Voor afnemers die het informatiemodel minder goed kennen, is dit het niet vanzelfspreken dat een vbo wel een relatie heeft met een pand, maar een pand niet met een vbo.  
Misschien is het daarom goed om hier 'geen relatie met andere resources conform het informatiemodel BAG' van te maken.